docs/src/concrete-playback.md

```diff
@@ -0,0 +1,59 @@
+# Concretely playing back proof harnesses
+
+When the result of a certain check comes back as a `FAILURE`,
+this _experimental_ feature allows users to debug their proof harness and understand what
+caused the failing check.
+
+## Setup
+
+The user needs to have a fairly recent version of the Kani source code somewhere on their computer.
```

Comment:
Be specific. This will be added on version 0.9.

Reply:
done